Public/Src/Cache/ContentStore/Vfs/VfsCasConfiguration.cs

```diff
@@ -42,6 +42,26 @@ public class VfsCasConfiguration
         /// </summary>
         public AbsolutePath VfsRootPath { get; }
 
+        /// <summary>
+        /// The root of mounted virtualized directories
+        /// </summary>
+        public AbsolutePath VfsMountRootPath { get; }

```

Comment:
>VfsMountRootPath [](start = 28, length = 16)

Perhaps more comment on the difference between `VfsMountRootPath`, `VfsMountRelativeRoot` (why is it needed?), and `VfsRootPath`.